packages/core/src/hooks/useTokenBalance.ts

```diff
@@ -0,0 +1,11 @@
+import { Interface } from '@ethersproject/abi'
+import { BigNumber } from '@ethersproject/bignumber'
+import { Falsy } from '../model/types'
+import { useContractCall } from './useContractCall'
+
+const TokenABI = new Interface(['function balanceOf (address) view returns (uint256)'])
```

Comment:
Let's use full ABI file instead (it quickly get messy), like `src/constans/abis/erc20.json`